src/styles/blocks/snake.scss

```
@@ -0,0 +1,30 @@
+.snake {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));

```

```suggestion
  grid-template-columns: repeat(auto-fill, minmax($variable, 1fr));

```
300px better to make a variable. Its use more than one time.